android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java

```diff
@@ -0,0 +1,13 @@
+package com.expensify.chat.generated;
+
+import java.util.Arrays;
+import java.util.List;
+import org.unimodules.core.interfaces.Package;
+
+public class BasePackageList {
+  public List<Package> getPackageList() {
+    return Arrays.<Package>asList(
+        new expo.modules.haptics.HapticsPackage()
+    );
+  }
+}
```

Comment:
Is this file meant to be included? I know I see these files sometimes generated when I run the android emulator, I'm not  sure if they're meant to be committed. They are a bit of a mystery to me.

Reply:
I've asked about it here: https://expensify.slack.com/archives/C01GTK53T8Q/p1628605049158100

Reply:
No, it was not meant to ge included, I added this accidently. I will remove it just in case!

Reply:
Yeah I think that's the move. If we decide on a solution for it in the thread, we can add it back or gitignore it in another PR.